webpack.config.ts

```diff
@@ -116,7 +116,6 @@ const config: Configuration = {
 			),
 
 			__filebasename: webpack.DefinePlugin.runtimeValue(
-				// @ts-expect-error
```

Comment:
Let's keep unrelated package-lock changes for the next lint. I added some lines in #4111 because they are macOS-specific and always get added by `npm i`, just for macOS users :(

Reply:
I am doing it now. I was just so mad I wanted to just commit and fix it up.